crates/air-lib/interpreter-data/src/value.rs

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2023 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use crate::JValue;
+
+use serde::{Deserialize, Serialize};
+
+use std::cell::RefCell;
+use std::rc::Rc;
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+#[serde(transparent)]
+pub struct RawValue {
+    raw: Box<str>,
+
+    #[serde(skip)]
+    parsed: RefCell<Option<Rc<JValue>>>,
```

Comment:
looks like `Cell` is enough here

Reply:
It seems one cannot clone `Cell` contents directly, at least with stable API.